CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Changes since v2.12
 - OIDC access tokens are now revoked on logout if the OIDC server provides a `revocation_endpoint`. (#2176)
 - Application attachment fields now accept multiple attachments. (#2122)
 - It's now possible to add a text to the login page after the login button using extra translations (:t.login/intro2) (#2214)
+- It is possible to sort applications based on the application external id accumulated number (#2183)
```

Comment:
It was possible before? This should say what it actually does (i.e. tries to sort them correctly).

Reply:
right, should have formulated more specifically.